events/12th-annual-users-meeting-2017.html

```diff
@@ -249,7 +249,7 @@ <h3 class="event-title">Workshops - Round 2</h3>
                         Montpellier)</li>
                 </ul>
                 <ul class="event-workshop">
-                    <li class="event-workshop-title"><span class="event-workshop-room">Room 3</span> - <a href="https://www.upf.edu/web/physense/rocket-platform" target="_blank">Rocket</a>: a flexible, modular, cloud-based platform for the management of heterogeneous information from acquisitions and tools for biomedical research (Chong Zhang, Neubias)</li>
+                    <li class="event-workshop-title"><span class="event-workshop-room">Room 3</span> - <a href="https://www.upf.edu/web/physense" target="_blank">Rocket</a>: a flexible, modular, cloud-based platform for the management of heterogeneous information from acquisitions and tools for biomedical research (Chong Zhang, Neubias)</li>
```

Comment:
commit not needed.
We should roll it back so when they fix the access issue it should work

Reply:
Removed the commit. Only the ignoring of the website by linkchecker is now in.